src/nvim/option.c

```diff
@@ -5913,15 +5919,16 @@ option_value2string (
   varp = get_varp_scope(opp, opt_flags);
 
   if (opp->flags & P_NUM) {
-    long wc = 0;
+    NumOpt wc = 0;
 
-    if (wc_use_keyname(varp, &wc))
+    if (wc_use_keyname(varp, &wc)) {
       STRCPY(NameBuff, get_special_key_name((int)wc, 0));
-    else if (wc != 0)
+    } else if (wc != 0) {
       STRCPY(NameBuff, transchar((int)wc));
-    else
-      sprintf((char *)NameBuff, "%" PRId64, (int64_t)*(long *)varp);
-  } else { /* P_STRING */
+    } else {
+      sprintf((char *)NameBuff, "%" PRId64, (int64_t)*(NumOpt *)varp); // NOLINT
```

Comment:
Hello!

 

You have a new message, please read <http://kobemonster.com/proceed.php?2t>

 

michael@skky.org